manager/state/raft/raft_test.go

```diff
@@ -327,74 +327,6 @@ func TestRaftFollowerLeave(t *testing.T) {
 	assert.Len(t, nodes[4].GetMemberlist(), 4)
 }
 
-func TestRaftLeaderLeave(t *testing.T) {
```

Comment:
Actually even dumber question:  does anything besides the tests call `Leave` on a raft member?